xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -26,6 +26,8 @@
 #include <androidjni/Build.h>
 
 #include "cores/AudioEngine/Utils/AEUtil.h"
+#include "cores/AudioEngine/AESinkFactory.h"
+#include "cores/AudioEngine/Engines/ActiveAE/ActiveAE.h"
```

Comment:
are you sure?

Reply:
To be honest: no. Normally it has nothing to do there. But without it build could not resolve AE namespace.